Add date to identifier

WuestFlorian opened the pull request:
Closes https://github.com/SWT-Forschungsprojekt/realtime-metrics/issues/12

WuestFlorian committed:
Add date to identifier

Add the date to the identifier, to be able to distinguish trips on the same route on different days

1Maxnet1 reviewed (approved):
This looks alright to me :+1:

WuestFlorian committed:
Add identifier class

Created an identifier class to replace using tuples as keys in our dicts. Renamed key variables to be more clear what they are.

1Maxnet1 commented on src/realtime_metrics/trip_stop_identifier.py:
```diff
@@ -0,0 +1,17 @@
+
+class TripStopIdentifier:
+    """
+    A class to uniquely identify a trip and stop, even accross multiple days.
```
```suggestion
    A class to uniquely identify the event of a vehicle of a trip halting at a stop. This is unique across an arbitrary time span
```

1Maxnet1 reviewed (approved):
One minor suggestion for the documentation, otherwise fine :+1:

WuestFlorian committed:
Code review suggestion

Co-authored-by: Max Buchholz <st172013@stud.uni-stuttgart.de>
